aldryn_events/forms.py

```diff
@@ -169,7 +169,9 @@ def clean(self):
         # to reverse, in case of success that would mean that the app_config
         # is correct and can be used.
         data = super(AppConfigPluginFormMixin, self).clean()
-        if not namespace_is_apphooked(data['app_config'].namespace):
+        app_config = data.get('app_config', None)
```

Comment:
Real error just says ``Select a valid choice. That choice is not one of the available choices.`` but we show more informative error (though the original error will still be present).